tests/e2e/vmgroups_test.go

```diff
@@ -113,7 +113,7 @@ func (vg *VmGroupTest) TearDownSuite(c *C) {
 
 	cmd := adminconst.ListVMgroups
 	out, err = ssh.InvokeCommand(vg.config.EsxHost, cmd)
-	log.Printf(out)
+	// log.Printf(out)
```

Comment:
This is how it looks like on CI console. I just feel it unnecessarily uses up the log space.
```
2017/08/11 02:12:54 stty: standard input: Inappropriate ioctl for device
Uuid            Name           Description      Default_datas..  VM_list          
--------------  -------------  ---------------  ---------------  ---------------  
11111111-111..  _DEFAULT       This is a def..  _VM_DS                            
e3290eef-591..  vmgroup_test1                   datastore1       photon.vmfs,U..  
ca6e7108-704..  vmgroup_test2                   vsandatastore
```
And we can always enable it for debug purposes.